### PR TITLE
Created the bubble sort sequence generator

### PR DESCRIPTION
**Pull request by iRustom**
Completed the sequence generator, along with several test cases to test its correctness. The test cases display each step of the sorting algorithm and what indices were swapped in the step.

### CODE REVIEW

**Review by OmarElfouly — CHANGES_REQUESTED** [review]
Include examples of expected and actual outputs in the bottom of your class

**Comment by iRustom** [comment]
> Include examples of expected and actual outputs in the bottom of your class

Included the original array with the outputs of each step as comments at the bottom of my file.

File: app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.scrumptious.algorithmvisualizer
+
+fun bubbleSortSequenceGenerator(inputArr: Array<Int>): Array<Pair<Array<Int>, Array<Int>>> {
```
Comment (OmarElfouly) [inline app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt:3]: Consider making this a class at some point

File: app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    )
+
+    for (inputArr in inputArrays) {
+        val stepsAndSwaps = bubbleSortSequenceGenerator(inputArr)
+        displaySteps(inputArr, stepsAndSwaps)
+        println()
+    }
+}
+
+Test Cases:
```
Comment (OmarElfouly) [inline app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt:55]: Looks good!

**Review by OmarElfouly — COMMENTED** [review]
You can either choose to make this a class now or make it a class later in development.

File: app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt (hunk 1 above)
Comment (iRustom) [reply app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt]: I have created the class and placed the function within it.

File: app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            }
+            if (!swapped) {
+                break
+            }
+        }
+    }
+}
+
+
+fun displaySteps(inputArr: Array<Int>, steps: MutableList<Pair<Array<Int>, Array<Int>>>) {
```
Comment (OmarElfouly) [inline app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt:30]: Please comment this.

**Review by OmarElfouly — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt (hunk 3 above)
Comment (iRustom) [reply app/src/main/java/com/scrumptious/algorithmvisualizer/BubbleSortSequenceGenerator.kt:30]: Added.

**Review by OmarElfouly — APPROVED** [review]
Looks great!

**Pull request merged**